CHANGELOG.md

```diff
@@ -21,102 +21,224 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 
```

Comment:
^^^
I can't mark the line, but the line above this:
```suggestion
See the documentation on [API stability](https://docs.seqan.de/seqan/3.0.1/about_api.html) to learn about
```

Reply:
great thanks. I also included a 3.0.0 documentation and added the index file so both 3.0.0 and 3.0.1 are on it

Reply:
Is the learn about ... intended or an unfinished sentence?

Reply:
Shouldn't we link to the most recent API stability though? In case we change something in the future?